Week2/homework/maartjes-work.js

```diff
@@ -47,15 +47,16 @@ const maartjesHourlyRate = 20;
 
 function computeEarnings(tasks, hourlyRate) {
   // Replace this comment and the next line with your code
-  console.log(tasks, hourlyRate);
+  const hours = tasks.map(task => task.duration / 60);
+  return hours.filter(hour => hour >= 2).reduce((sum, hour) => sum + hour * hourlyRate, 0);
 }
```

Comment:
With the parameter names you chose for the array methods you clearly demonstrate your understand what happens to the data as it get processed while traversing the method chain. Good show! But you can combine this all in one big chain like this:

```js
function computeEarnings(tasks, hourlyRate) {
  return tasks
    .map(task => task.duration / 60)
    .filter(hour => hour >= 2)
    .reduce((sum, hour) => sum + hour * hourlyRate, 0);
}
```